Standards/scs-0400-v1-status-page-create-decision.md

```diff
@@ -153,7 +153,7 @@ With those requirements in mind the projects that initially were found, were eva
 | user management | ✅ | ❌ | ❌ | ❌ | ✅ by OIDC | ⁇ through github? | ❌ |
 | different output format on notification | ❌ | ❌ | ✅ | ✅ | ✅ | ❌ | ✅ |
 | external hosting | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ looks like you are limited to github | ✅ |
-| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abondend by the owner |
+| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abndend by the owner |
```

Comment:
```suggestion
| project healthy | ❌ last commit 17 months | ❌ last commit 3 years | ❌ last commit 5 months | ✅ last commit 2 months | ✅ recent activities | ✅ recent activities | ❌ archived and abandoned by the owner |
```

Reply:
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b